src/components/ha-form/ha-form-expandable.ts

```diff
@@ -0,0 +1,83 @@
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import type { HomeAssistant } from "../../types";
+import "./ha-form";
+import type {
+  HaFormDataContainer,
+  HaFormElement,
+  HaFormExpandableSchema,
+  HaFormSchema,
+} from "./types";
+
+@customElement("ha-form-expandable")
+export class HaFormExpendable extends LitElement implements HaFormElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public data!: HaFormDataContainer;
+
+  @property({ attribute: false }) public schema!: HaFormExpandableSchema;
+
+  @property({ type: Boolean }) public disabled = false;
+
+  @property() public computeLabel?: (
+    schema: HaFormSchema,
+    data?: HaFormDataContainer
+  ) => string;
+
+  @property() public computeHelper?: (schema: HaFormSchema) => string;
+
+  protected render(): TemplateResult {
+    return html`
+      <ha-expansion-panel outlined .expanded=${Boolean(this.schema.expanded)}>
+        <div slot="header">
```

Comment:
This should be a heading tag instead of a `div`, although setting the correct level depends on the use case.  Maybe set `role="heading" aria-level=${this.schema.level}` and default the level to 2 or 3?

Reply:
Yes, level depends on the context, it must be option in the schema. 
What's the best default : no level, 2 or 3 ?

Reply:
It should have one. Most dialogues have a top heading at level two, so probably three is the best default.